app.riftshare.RiftShare.yaml

```
@@ -0,0 +1,35 @@
+app-id: app.riftshare.RiftShare
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+build-options:
+  append-path: /usr/lib/sdk/golang/bin
+  env: 
+    - GOROOT=/usr/lib/sdk/golang
+command: RiftShare
+finish-args:
+  # Window Manager
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--device=dri'
+  - '--share=network'
+  - '--filesystem=home'
+  - '--talk-name=org.freedesktop.Notifications'
+modules:
+  - name: riftshare
+    sources:
+      - type: archive
+        url: "https://github.com/achhabra2/riftshare/releases/download/v0.1.7/riftshare-vendored.tar.gz"
+        sha256: bc1f55dabc376b9c73d0f0f3386fb48ca5fc9a354eb5d178e1178b30e948ba1e
+        strip-components: 1
+    buildsystem: simple
+    build-commands:
+      - go build -tags desktop,production -ldflags "-w -s" -o RiftShare
+      - install -Dm755 RiftShare /app/bin/RiftShare
+      - install -Dm644 build/linux/app.riftshare.RiftShare.desktop /app/share/applications/app.riftshare.RiftShare.desktop
+      - install -Dm644 build/linux/app.riftshare.RiftShare.png /app/share/icons/hicolor/512x512/apps/app.riftshare.RiftShare.png
+      - install -Dm644 build/linux/app.riftshare.RiftShare.appdata.xml /app/share/metainfo/app.riftshare.RiftShare.appdata.xml
+      - install -Dm644 build/linux/app.riftshare.RiftShare.appdata.xml /app/share/appdata/app.riftshare.RiftShare.appdata.xml
```

```suggestion
      - install -Dm644 build/linux/app.riftshare.RiftShare.appdata.xml /app/share/appdata/app.riftshare.RiftShare.appdata.xml
```

you only need it once